app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
```

Comment:
What is the disadvantage of leaving it? Understood that we may not have a need for it right now according to our wireframes, however, this is a basic aspect of many applications and it may be useful in the future.